kore/src/Control/Monad/Unify.hs

```diff
@@ -0,0 +1,106 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+
+module Control.Monad.Unify where
+
+import           Colog
```

Comment:
If we need this here, let's export it from `Kore.Logger` instead; I would like that module to be the "single source of truth" for logging-related items.